tests/conftest.py

```diff
@@ -93,6 +101,67 @@ def create_client(self, title=None):
         self.client_procs.append(proc)
         return winid
 
+    def complete(self, cmd, partial=False, position=None):
+        """
+        Return a sorted list of all completions for the next argument for the
+        given command, if position=None. If position is given, then the argument
+        of the given position is completed.
+
+        Set 'partial' if some of the completions for the given command are
+        partial. If not in 'partial' mode, trailing spaces are stripped.
+        """
+        args = self._possibly_split_command(cmd)
+        if position is None:
+            position = len(args)
+        proc = self.call(['complete_shell', position] + args)
+        items = [ ]
+        for i in proc.stdout.splitlines(False):
+            if partial:
+                items.append(i)
+            else:
+                if not i.endswith(' '):
+                    raise Exception(("completion for \"{}\" returned the partial "
+                                    + "completion item \"{}\"").format(cmd, i)
+                                    ) from None
+                else:
+                    items.append(i[0:-1])
+        return sorted(items)
+
+    def list_children_via_attr(self, object_path):
+        """
+        list the names of children of the
```

Comment:
Start sentence with capital letter ;-)

Reply:
These and the `_possibly…` thing fixed in 8bd78b8